### PR TITLE
Add a session variable to show or hide hidden columns

### LINKED ISSUE #4509: Add a session variable to show or hide hidden columns
Sometimes it is necessary to display the value from the hidden column to help debug, such as determining whether the result of batch deletion of imported data is correct, whether the result of compaction is correct, etc. In the current implementation, this information is completely hidden and cannot be viewed. Therefore, you need to add a session variable `show_hidden_columns` to control whether to show hidden columns and filter hidden column data.
When `show_hidden_columns` is true, functions that get the table's schema such as get_schema will not return hidden columns at the beginning, and the filter condition of `__DELETE_SIGN__=true` will be added to olap sacanNode. When `show_hidden_columns` is false, the opposite is true

### PR DESCRIPTION
**Pull request by yangzhg**
## Proposed changes

Sometimes we need to show hidden columns for debug.
So we need to add a session variable to show or hide hidden columns

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] Documentation Update (if none of the other choices apply)
- [] Code refactor (Modify the code structure, format the code, etc...)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have create an issue on (Fix #4509), and have described the bug/feature there in detail
- [x] Compiling and unit tests pass locally with my changes
- [] I have added tests that prove my fix is effective or that my feature works
- [x] If this change need a document change, I have updated the document
- [] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/zh-CN/administrator-guide/load-data/batch-delete-manual.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+)
+PROPERTIES
+(
+    "timeout" = "3600"
+    
+);
+
+```
+
+#### reoutine load
```
Comment (EmmyMiao87) [inline docs/zh-CN/administrator-guide/load-data/batch-delete-manual.md:98]:
```suggestion
#### routine load
```

File: docs/zh-CN/administrator-guide/load-data/batch-delete-manual.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+在新建表时如果 `enable_batch_delete_by_default` 为true 则新建表的都支持批量删除，如果为false 则新建的表默认不支持批量删除功能。
+对于一个不支持批量删除功能的表 如果项要使用批量删除 可以使用如下语句：
+`ALTER TABLE tablename ENABLE FEATURE "BATCH_DELETE"` 来启用批量删除。
+如果确定一个表是否支持批量删除，可以通过 设置一个session variable 来显示隐藏列 `SET show_hidden_columns=true` ，之后使用`desc tablename`，如果输出中有`__DELETE_SIGN__` 列则支持，如果没有则不支持
+
+## 注意
+1. 由于除stream load 外的导入操作在doris 内部有可能乱序执行，因此在使用`MERGE` 方式导入时如果不是stream load，需要与 load sequence 一起使用，具体的 语法可以参照sequence列 相关的文档 
+2. `DELETE ON` 条件只能与 MERGE 一起使用
+
+## 使用示例
```
Comment (EmmyMiao87) [inline docs/zh-CN/administrator-guide/load-data/batch-delete-manual.md:135]: 是不是可以考虑把使用的流程都展示一下。比如当前需求是什么？然后导入前数据是什么样子的，怎么导入，最后导入后的变化是什么？以及推荐的使用场景？

File: fe/fe-core/src/main/java/org/apache/doris/catalog/SchemaTable.java [hunk 3]
```diff
@@ -76,7 +76,7 @@ public static Builder builder() {
                             "tables",
                             TableType.SCHEMA,
                             builder()
-                                    .column("TABLE_CATALOG", ScalarType.createVarchar(FN_REFLEN))
+                                    .column(" TABLE_CATALOG", ScalarType.createVarchar(FN_REFLEN))
```
Comment (EmmyMiao87) [inline fe/fe-core/src/main/java/org/apache/doris/catalog/SchemaTable.java:79]: Without space?

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**